server/routes/post.route.js

```diff
@@ -27,6 +29,12 @@ router.route('/:postId/favorite')
 router.route('/:postId/unfavorite')
   .post(expressJwt({ secret: config.jwtSecret }), favoriteCtrl.unfavorite);
 
+router.route('/:postId/listened')
+  .post(expressJwt({ secret: config.jwtSecret }), listenedCtrl.create);
+
+router.route('/:postId/listened')
```

Comment:
This would make user's listen history public. My initial though it yes. But, I'm not sure people would like this in general?

Reply:
Good point, I can remove it for now.